src/lib/outros.json

```
@@ -164,6 +164,16 @@
             ]
         }
     ],
+    "SDF": [
+        {
+            "acronym": "SDF",
+            "meaning": "A SDF(Semana do Fera) é um evento realizado na primeira semana no curso de Computação todos os periodos, com o intuito de dar boas-vindas aos novos estudantes(feras).",
```

Apenas uma correção besta 😄 
```suggestion
            "meaning": "A SDF(Semana do Fera) é um evento realizado na primeira semana no curso de Computação todos os períodos, com o intuito de dar boas-vindas aos novos estudantes(feras).",
```